README.md

```diff
@@ -102,6 +103,7 @@ Awesome list of React components with render props and resources.
 - [react-pro-metronome](https://github.com/rigobauer/react-pro-metronome): React component for creating your own metronome
 - [react-sentinel](https://github.com/YurkaninRyan/react-sentinel): Abstracts away requestAnimationFrame, allowing you to poll anything for props
 - [react-t-minus](https://github.com/jxom/react-t-minus): A React component to handle simple countdowns.
+- [react-queue](https://github.com/theKashey/react-queue): A declarative task queue.
```

Comment:
Should go above `react-sentinel` 👍

Reply:
I was absolutely sure Q goes after T. Don't worry, I've got a primary school nearby.